languages/wikipron/src/scrape.py

```diff
@@ -62,13 +60,19 @@ def main():
         languages = json.load(source)
     # "2020-01-15" (Big Scrape 3)
     cut_off_date = datetime.date.today().isoformat()
+    valid_keys = ["casefold", "no_skip_spaces_pron", "no_skip_spaces_ortho"]
     for iso639_code in languages:
+        wikipron_accepted_settings = {
```

Comment:
This makes the same lookup (`languages[iso639]`) twice. Can you rewrite this as a loop to avoid that?

Reply:
Let me know if you think my solution for this is alright. It uses the least amount of lookups I could come up with, but does entail looping through all of each sub-dictionary for every languages.